packages/wallets/src/components/OptionsAndMultipliersListing/OptionsAndMultipliersListing.tsx

```diff
@@ -62,16 +64,20 @@ const OptionsAndMultipliersListing = () => {
                             Earn a range of payouts by correctly predicting market price movements with{' '}
                             <a
                                 className='wallets-options-and-multipliers-listing__header-subtitle__link'
-                                href='#'
+                                href='https://deriv.com/trade-types/options/'
```

Comment:
This page is not exist in the deriv.com anymore, use below one

```suggestion
                                href='https://deriv.com/trade-types/options/digital-options/up-and-down/'
```